core/templates/components/forms/custom-forms-directives/edit-thumbnail-modal.component.html

```diff
@@ -26,11 +26,11 @@ <h3>Upload Thumbnail</h3>
       <div class="oppia-form-error" *ngIf="invalidFilenameWarningIsShown">
         Please upload a valid filename image i.e; The svg file must not start with a dot or have '/' or 2 consecutive dots.
       </div>
-      <div class="oppia-form-error" *ngIf="invalidTagsAndAttributes?.tags.length">
-        Unsupported SVG tags found: {{ invalidTagsAndAttributes.tags }}
+      <div class="oppia-form-error oppia-unsupported-svg-tag-attribute-error" *ngIf="invalidTagsAndAttributes?.tags.length">
+        Note: Removed {{ invalidTagsAndAttributes.tags }} tag. If your file looks distorted please <a href="https://github.com/oppia/oppia/issues/new?title=Image%20file%20got%20distorted%20on%20upload&body=Image%20file%20got%20distorted%20while%20uploading%20on%20%7B%7BADDRESS_URL_HERE%7D%7D.%0A%0AThe%20Image%20file%20is%20attached%20below%20-%0A%0A%7B%7BIMAGE_HERE%7D%7D%0A%0AScreenshots%20of%20the%20problem-%0A%0A%7B%7BSCREENSHOTS_HERE%7D%7D" target="_blank" rel="noopener">let us know</a>, then try uploading a different SVG.
       </div>
-      <div class="oppia-form-error" *ngIf="invalidTagsAndAttributes?.attrs.length">
-        Unsupported SVG attributes found: {{ invalidTagsAndAttributes.attrs }}
+      <div class="oppia-form-error oppia-unsupported-svg-tag-attribute-error" *ngIf="invalidTagsAndAttributes?.attrs.length">
+        Note: Removed {{ invalidTagsAndAttributes.attrs }} attribute. If your file looks distorted please <a href="https://github.com/oppia/oppia/issues/new?title=Image%20file%20got%20distorted%20on%20upload&body=Image%20file%20got%20distorted%20while%20uploading%20on%20%7B%7BADDRESS_URL_HERE%7D%7D.%0A%0AThe%20Image%20file%20is%20attached%20below%20-%0A%0A%7B%7BIMAGE_HERE%7D%7D%0A%0AScreenshots%20of%20the%20problem-%0A%0A%7B%7BSCREENSHOTS_HERE%7D%7D" target="_blank" rel="noopener">let us know</a>, then try uploading a different SVG.
```

Comment:
What happens when there are several attributes as well as tags that are bad? In such cases, it might be nice to show just one warning message.

Reply:
For the issue template, can we make it looks like this?
![image](https://user-images.githubusercontent.com/11008603/155921096-12e697be-f16d-42a3-8e3c-bde9a3ba826b.png)


Minor wording changes and I've added a section where we auto-fill the bad tags and attributes (to make it easier to reproduce).

Reply:
Done!